### ISSUE TITLE
Autocomplete from different files

### ISSUE THREAD

**Issue by PixelT** [marijnh/tern_for_sublime#87]
For example:
I have folder /js/ with 2 files:
- function1.js
- function2.js

In function1.js I defined an object:
```
testObject = {
	'prop1': 'val1',
	'prop2': 'val2',
	'prop3': function(val3, val4) {
		return val3 * val4;
	}
}
```

When I write in function2.js:
```
testObject.
```
the autocomplete / hints for this object doesn't show... (also the testObject is unavailable in the hint)

What I must done, to 'scan' all *.js files or / and all files defined in specific catalog?
My .tern-project settings is only:
```
{
	"libs": [
		"browser",
		"jquery"
}
```
I try add loadEagerly
```
{
	"libs": [
		"browser",
		"jquery"
	],
	"loadEagerly": [
		"js/*.js"
	]
}
```
but it doesn't change :(

**Comment by marijnh** [comment]
If I recreate the situation you describe, I do get completions if I open function2.js and press alt-/ after the dot.

**Comment by PixelT** [comment]
Yes - when I create this files, the autocomplete will be work - OK
But when I close SublimeText and open in later (or just restart ST), the autocomplete with different files (which I describe in 1 post) will not be works

**Comment by marijnh** [comment]
Okay, but that's what I did as well (close function1.js tab, close ST, reopen ST, complete in function2.js)
